docs/developing.md

```diff
@@ -0,0 +1,60 @@
+Quickstart for developing Navigator
+===================================
+
+Setting up
+----------
+
+Install minikube and start a cluster with RBAC enabled:
+
+    minikube start --extra-config=apiserver.Authorization.Mode=RBAC
+
+Work around `kube-dns` and other services having problems when RBAC is enabled in minikube:
+
+    kubectl create clusterrolebinding cluster-admin:kube-system \
+        --clusterrole=cluster-admin \
+        --serviceaccount=kube-system:default
+
+Fetch the docker configuration:
+
+    eval (minikube docker-env)
+
+Build images in minikube's docker:
+
+    make BUILD_TAG=dev all
+
+Or quicker (skips tests):
+
+    make BUILD_TAG=dev build docker_build
+
+Install helm into the minikube cluster:
+
+    helm init
+
+Install navigator using the helm chart:
+
+    helm install contrib/charts/navigator \
+        --set apiserver.image.pullPolicy=IfNotPresent \
+        --set apiserver.image.tag=dev \
+        --set controller.image.pullPolicy=IfNotPresent \
+        --set controller.image.tag=dev \
+        --name navigator --namespace navigator --wait
+
+Test navigator is deployed by creating a demo elasticsearch cluster:
+
+    kubectl create -f docs/quick-start/es-cluster-demo.yaml
```

Comment:
Worth adding here that you should set the pilot image tag to `dev`, and set pull policy to Never as well.

Reply:
:+1: